package.json

```diff
@@ -1,47 +1,44 @@
 {
-  "name": "external",
-  "version": "0.0.1",
-  "description": "External Sites for use with Nextcloud",
-  "private": true,
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "eslint": "npx eslint --ext .js,.vue src",
-    "eslint:fix": "npx eslint --ext .js,.vue src --fix",
-    "build": "NODE_ENV=production npx webpack --progress",
-    "dev": "npx webpack --progress",
-    "watch": "npx webpack --progress --watch"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/nextcloud/external.git"
-  },
-  "author": "Joas Schilling",
-  "license": "AGPL-3.0-or-later",
-  "bugs": {
-    "url": "https://github.com/nextcloud/external/issues"
-  },
-  "homepage": "https://github.com/nextcloud/external#readme",
-  "devDependencies": {
-    "@babel/core": "^7.10.2",
-    "@babel/preset-env": "^7.10.2",
-    "@nextcloud/browserslist-config": "^1.0.0",
-    "@nextcloud/eslint-config": "^2.0.0",
-    "@nextcloud/eslint-plugin": "^1.4.0",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.1.0",
-    "eslint": "^7.1.0",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-standard": "^4.0.1",
-    "eslint-plugin-vue": "^6.2.2",
-    "standard": "^14.3.4",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11"
-  },
-  "dependencies": {
-    "@nextcloud/router": "^1.1.0",
-    "escape-html": "^1.0.3"
-  }
+	"name": "external",
+	"version": "3.7.0",
+	"private": true,
+	"description": "",
+	"author": "Joas Schilling <coding@schilljs.com>",
+	"scripts": {
+		"test:unit": "echo \"Error: no test specified\" && exit 1",
+		"lint": "eslint --ext .js,.vue src",
+		"lint:fix": "eslint --ext .js,.vue src --fix",
+		"build": "NODE_ENV=production npx webpack --progress",
+		"dev": "npx webpack --progress",
+		"watch": "npx webpack --progress --watch"
+	},
+	"dependencies": {
+		"@nextcloud/router": "^1.1.0",
+		"escape-html": "^1.0.3"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.10.2",
+		"@babel/preset-env": "^7.10.2",
+		"@nextcloud/browserslist-config": "^1.0.0",
+		"@nextcloud/eslint-config": "^2.0.0",
+		"@nextcloud/eslint-plugin": "^1.4.0",
+		"babel-eslint": "^10.1.0",
+		"babel-loader": "^8.1.0",
+		"eslint": "^7.1.0",
+		"eslint-plugin-import": "^2.20.2",
+		"eslint-plugin-node": "^11.1.0",
+		"eslint-plugin-promise": "^4.2.1",
+		"eslint-plugin-standard": "^4.0.1",
+		"eslint-plugin-vue": "^6.2.2",
+		"standard": "^14.3.4",
+		"webpack": "^4.43.0",
+		"webpack-cli": "^3.3.11"
+	},
+	"browserslist": [
+		"extends @nextcloud/browserslist-config"
+	],
+	"engines": {
+		"node": ">=10.0.0"
+	},
+	"license": "agpl"
```

Comment:
I believe, according to the docs, license field is supposed to be an SPDX identifier which is our case should be `AGPL-3.0-or-later` so this probably should stay the same.

https://docs.npmjs.com/files/package.json#license

Reply:
Right